src/shared_modules/utils/sqliteWrapperTemp.h

```diff
@@ -465,6 +466,12 @@ namespace SQLite
                 checkSqliteResult(result, sqlite3_errmsg(m_connection->db().get()));
                 ++m_bindParametersIndex;
             }
+            void bind(const int32_t index, const uint32_t value)
```

Comment:
Since the fix is on the manager side and we see that it only affects the MTU, I would remove this change to avoid edge conditions.